_docs/NumberInput/NumberInput.md

```diff
@@ -0,0 +1,96 @@
+# NumberInput Component
+
+**NumberInput** is a form-element component that enables inputting values of the type number. The values can be confined to a specific range, and the "step" (or increment) between the values can be defined.
+
+![NumberInput examples](./assets/numberInput.png)
+
+The NumberInput component improves upon the native `<input type="number">`:
+
+* provides ability to customize the stepper arrows design
+* provides a common React+Typescript API
+* works out the kinks of native implementations
+
+## Component API
+
+> tbd :: add method definition
+
+### Component Props
+
+| name        | type                                  | defaultValue | isRequired | description                              |
+| ----------- | ------------------------------------- | ------------ | ---------- | ---------------------------------------- |
+| value       | number |              | yes        | Used to set and change the value of the input |
+| placeholder | string  |              |            | Text to display if the value is null.     |
+| min         | number  | 1            |            |                                          |
+| max         | number  | 100          |            |                                          |
```

Comment:
I think we made an error here, and we have to remove that defaults, as well as you here.
https://github.com/wix/stylable-components/issues/170

if user didn't specify min/max there should not be limits. 
so no need to have default value